dso-l2/src/main/java/com/tc/services/EntityMessengerProvider.java

```diff
@@ -79,4 +80,9 @@ public void setMessageSink(Sink<VoltronEntityMessage> messageSink) {
     Assert.assertNotNull(messageSink);
     this.messageSink = messageSink;
   }
+
+  @Override
+  public void dumpStateTo(final StateDumper stateDumper) {
+    //TODO: implement this
```

Comment:
It is good to dump state information from EntityMessengerService such as retirementHandles, we need to keep a list of existing EntityMessengerService but when do we clean up these references?